static/js/base.js

```diff
@@ -219,7 +219,16 @@ function doneLoading (selector) {
 // Other goodies
 
 function formatDate (dateString) {
-    return (new Date(dateString + ':00')).toLocaleString(
+    dateString = dateString + ':00';
+        
+    // Replaces the first instance of a space,
+    //  and replace with "T" to fit ISO 8601,
+    //  which all modern browsers support
+    // Example: YYYY-MM-DD HH:MM:SS-TZ:tz --> YYYY-MM-DDTHH:MM:SS-TZ:tz
+    //  Note: TZ:tz is Hour and Minutes of Time Zone
+    dateString = dateString.replace(' ', 'T');
```

Comment:
```suggestion
    // Replaces the first instance of a space,
    //  and replace with "T" to fit ISO 8601,
    //  which all modern browsers support
    // Example: YYYY-MM-DD HH:MM:SS-TZ:tz --> YYYY-MM-DDTHH:MM:SS-TZ:tz
    //  Note: TZ:tz is Hour and Minutes of Time Zone
    dateString = `${dateString}:00`.replace(' ', 'T');
```

I should very this, but I think this looks nicer. :)